pkg/types/credentials.go

```diff
@@ -22,24 +22,24 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"errors"
+	"github.com/Peripli/service-manager/pkg/httpclient"
 )
 
 // Basic basic credentials
 type Basic struct {
-	Username string `json:"username,omitempty"`
-	Password string `json:"password,omitempty"`
+	Username string `json:"username"`
+	Password string `json:"password"`
 }
 
 type TLS struct {
-	Certificate string `json:"client_certificate,omitempty"`
-	Key         string `json:"client_key,omitempty"`
```

Comment:
dosen't it means that when we return the broker (eg. on GET v2/service_brokers) the user will get empty strings on certificate and key?